### DISCUSSION TITLE
BlueJay watch looses connection

### DISCUSSION THREAD

**Discussion by ben80** [NightscoutFoundation/xDrip#2703] [General]
Hello,
i was asked in the FB xdrip group to start a discussion here.

I setup my new bluejay watch for my 8y son but today and yesterday (the first two days) it lost connections quite often and for longer times, e.g. for 2h or for 1h. It seems to receive BG values sometimes every few minutes and then a break for 15min.

My son is wearing the watch on the same arm, as the dexcom G6.
Bluejay is configured for the "non-phone-slot".  On the phone the BYODA app is running. And also sometimes xdrip is open but currently not used. The source is set to "collection disabled". When i received the watch i first tried to configure the watch via the xdrip app. But since i wasn't sure about the setup i disabled the bluejay functions in xdrip. I then configured the watch using its touch screen.

Before i bought the watch the bluejay support told me, that some of the shipped watches could have a poor antenna. This leads to connection drops and in that case they would provide another watch. Not sure, if thats what i am experiencing, though. I tried to contact the support (about 10 days ago) but didn't get a response until today. Maybe they are on vacation.

I really hope we can figure out some solution.
Nik

**Comment by Navid200** [comment]
Do you also have a Dexcom receiver?

Do you see the third image on this page showing medical slot and phone slot?
https://navid200.github.io/xDrip/docs/bj_GTS_stngs.html
You have selected the medical slot, the option on the left, right?

**Reply by ben80** [reply]
Yes , I used the left one. After I set the option I got the message "using transmitter non phone slot".
As you can see on the screenshot. The failure shown, results from the previous sensor, which failed. I haven't used the bluejay at that time, only XDrip. 

![PXL_20230223_202646285](https://user-images.githubusercontent.com/8597725/221379274-cb833ba3-7dd1-4e03-95a7-61b824c1064a.jpg)

**Reply by Navid200** [reply]
When did you take that picture?  The date is wrong.

**Reply by ben80** [reply]
Last thursday, when I set up the watch. Feb 23rd.
Should be correct.

**Reply by Navid200** [reply]
Do you have a Dexcom receiver?

**Reply by ben80** [reply]
No, only the phone and the watch

**Reply by ben80** [reply]
Should I try to reset to factory settings and configure the watch again?

**Reply by Navid200** [reply]
When I have contacted the business, they have been extremely responsive.  I am surprised.
Let me send them a message.

**Reply by ben80** [reply]
Ok thanks

**Reply by ben80** [reply]
> When I have contacted the business, they have been extremely responsive. I am surprised. Let me send them a message.

Any luck with contacting them?
In the last two days, we had no updated BG value for 55, 40, 36 and 20 minutes. These are just the ones i am aware of. Then again i have updates every 5-8 minutes. But also often 15minutes. Even that is too long. Yesterday morning the BG dropped down to 50 and the watched showed it 15minutes later only.

**Reply by Navid200** [reply]
I did not get a response either.
I'm not sure why.
Hopefully it, it's a temporary interruption.

**Reply by Navid200** [reply]
How much time do you have for returning it for a full refund?

**Reply by ben80** [reply]
I got a first response from the developer today. He was away last week.
I'll let you know what he can find out.

**Reply by Navid200** [reply]
That makes sense now.  
BlueJay is a unique product.  It would be a shame if support ends.  I'm glad it seems there was a reasonable explanation.
Thanks

**Reply by ben80** [reply]
Finally the watch is working and providing updated values every few minutes.
In the end, i reset the watch to factory settings and configured it newly with xdrip. 
I havent switched to BYODA as source for the smartphone, yet. I will try that next.
At least i know now the watch is working properly and really good.
Thanks for your help @Navid200 .

**Reply by Navid200** [reply]
Did support update the watch firmware?

**Reply by ben80** [reply]
No, should they?

**Reply by Navid200** [reply]
Can you tell me what your firmware version is?

**Reply by ben80** [reply]
I will when i am home tomorrow

**Reply by Navid200** [reply]
My firmware is 2321.
It's possible that your watch was released with that firmware already on it.

**Reply by ben80** [reply]
Mine is 2321 as well.
Is this the latest?

**Reply by Navid200** [reply]
Yes, I believe so.
